agent/php_execute.c

```diff
@@ -857,6 +855,9 @@ static nrframework_t nr_try_detect_framework(
       }
 
       nr_framework_log("detected framework", frameworks[i].framework_name);
+      nrl_verbosedebug(
+          NRL_FRAMEWORK, "framework '%s' detected with %s, which ends with %s",
+          frameworks[i].framework_name, filename, frameworks[i].file_to_check);
```

Comment:
I don't know if it's interesting to log this or not. I used it in tests to ensure the framework is detected with a file that I thought it would be detected.